card-flow.md

```diff
@@ -0,0 +1,57 @@
+# Project Management Boards
+
+Our team uses a web-based Kanban planning tool, such as Trello or AgileZen, to manage our projects and track our progress. The boards serve two purposes: to allow clients to see what's happening and give feedback on a project, and to allow us to manage our team, our tasks, and stay coordinated. Each project has its own board. The board shows us the current state of all the things we're working on in our project and where each task is in the lifecycle of development. Each task is represented by a card on the board, and as cards move through the development process, we move them to different columns on the board.
+
+Everything the team is working on for a project, whether a feature, a bug, or a chore, should be represented by a card. We can't track something if it's not on the board. If we're doing something that's not represented by a card, we should ask why.
+
+## Board Flow
+
+In Kanban, we add cards on the far left-most column that are to be worked on by the team. As cards are ready to be worked on and there's availability by the team, we move the cards to the right as they go through the steps of software development.  Each column represents the stages that a specific project has defined.  We will illustrate our common flow below but each project may modify this based on business conditions.
+
+### Where's the backlog or icebox columns?
+
+Some tools have a place to keep cards that don't 'fit' into the current iteration or cycle of work.  This concept doesn't exist in Kanban and it's been our experience that trying to mimic either backlogs or iceboxes lead to out-dated and wasteful cards that ultimately slow down the team.  Instead we have the client keep any extra work not quite ready to be tackled elsewhere.  This allows only 'fresh' ideas to enter the board to be worked on by the team in the next 1-2 weeks at most.
+
+### The "Next" Column
+Once cards are approved and ready for developers to begin working on them, we move them to the "Next" column. Cards in this column are our queue of work, and are waiting for someone to be assigned to them. A developer can assign themselves to a card when they are ready to begin work on it.
+
+### The "Development" Column
+
+The "Development" column is for cards that are actively being worked on. It is important that a developer has only one card in this column at a time. If they need to work on something else, this card can move back to ready so as not to let things build up or block. Our project board system follows the traditional Kanban pull based system pretty closely. Things can only move right if there's space for them, and a dev team can really only take on 3-4 cards at once in the development stage. More than that and things sit idle. The board should be an accurate reflection of the project at any point in time, so if a developer is not currently working on a card in the "Development" column, they can move it back so our progress is accurately reflected.
```

Comment:
oh, one minor thing to correct.  

...back to ready so... should change to ...back to "Next" so...  

We might revise our column names at some point but we'll want to make sure that we're consistent in how were refer to them.  I think the "<Column Name>" pattern seems best for this guide.